src/lib.rs

```diff
@@ -24,34 +24,46 @@ pub fn sync_remote_execute(cmd: &str) -> (i32, String, String) {
     use std::time::Duration;
     let very_long_timeout = Duration::new(60 * 60 * 24, 0); // 24h
 
-    let client = reqwest::blocking::Client::builder()
+    let client_result = reqwest::blocking::Client::builder()
         .timeout(very_long_timeout)
-        .build()
-        .unwrap(); // todo: remove unwrap
+        .build();
+    let client = match client_result {
+        Ok(c) => c,
+        Err(e) => {
+            eprintln!("sync_remote_execute: Failed to build client: {}", e);
+            return (
+                -3,
+                "".to_string(),
+                "sync_remote_execute API: Client Build Error (ERROR 910-21087-27552)".to_string(),
```

Comment:
We should also send the error too: 
```suggestion
format!("sync_remote_execute API: Client Build Error (ERROR 910-21087-27552). Error: {}" e)
```

Reply:
(Changed to:
format!("sync_remote_execute API: Client Build Error (ERROR 910-21087-27552). Error: {e}")
)